### PR TITLE
Support consistent SSO for migrated JSR 375 apps for issue #4227

### PR DESCRIPTION
**Pull request by teddyjtorres**
For issue #4227

### CODE REVIEW

**Comment by teddyjtorres** [comment]
#build

**Comment by teddyjtorres** [comment]
The errors were reviewed and are not related to the code being delivered. Another personal build was submitted for running com.ibm.ws.security.javaeesec_fat in full mode.

**Comment by teddyjtorres** [comment]
The full FAT tests for com.ibm.ws.security.javaeesec_fat completed successfully.

**Comment by teddyjtorres** [comment]
@OpenLiberty/message-reviewer Please review.

File: dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -33,6 +33,9 @@ ssoDomainNames.desc=A pipe (|) separated list of domain names that SSO Cookies s
 ssoUseDomainFromURL=Use domain name from the request URL
 ssoUseDomainFromURL.desc=Specifies whether to use the domain name from the request URL for the cookie domain.
 
+useSSOForJaspic=Enable single sign-on for JASPIC
+useSSOForJaspic.desc=Specifies whether single sign-on is enabled for a JASPIC authentication.
+
 ssoRequiresSSL=Single sign-on requires SSL
 ssoRequiresSSL.desc=Specifies whether a SSO cookie is sent over SSL. The equivalent property in the full application server profile is requiresSSL.
```
Comment (bsbyrd1) [inline dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties]: Suggestion:
Specifies whether an SSO cookie is sent over SSL. The equivalent property in the full application server profile is requiresSSL.

Comment (teddyjtorres) [reply dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties]: @bsbyrd1 Hi. Thank you for the review. There might have been a misunderstanding. The new property to review is useSSOForJaspic and the message reviewed was ssoRequiresSSL.

**Comment by teddyjtorres** [comment]
@OpenLiberty/message-reviewer @bsbyrd1  Please review. There were some changes to the useLtpaSSOForJaspic and useLtpaSSOForJaspic.desc messages that were proposed after the original review. Thank you for your help.

**Comment by teddyjtorres** [comment]
#build

**Comment by teddyjtorres** [comment]
#build

**Comment by teddyjtorres** [comment]
The failure in testFeaturesStart is not related to the code being delivered and there is already a defect for it. The failure in the webcontainer.security's Audit test was caused by the following problem in jsp and it is not related to the code being delivered,

[ERROR   ] CWWKF0029E: Could not resolve module: com.ibm.ws.jsp.factories [167]
  Unresolved requirement: Import-Package: org.apache.jasper.el; version="2.2.0"
    -> Export-Package: org.apache.jasper.el; bundle-symbolic-name="com.ibm.ws.jsp.jasper"; bundle-version="1.0.23.201810151424"; version="2.2.0"; thread-context="true"; uses:="com.ibm.websphere.ras.annotation,javax.el,javax.servlet.jsp.el"
       com.ibm.ws.jsp.jasper [168]
         No resolution report for the bundle.  Bundle was not resolved because of a uses contraint violation.
  org.osgi.service.resolver.ResolutionException: Uses constraint violation. Unable to resolve resource com.ibm.ws.jsp.factories [osgi.identity; osgi.identity="com.ibm.ws.jsp.factories"; type="osgi.bundle"; version:Version="1.0.23.201810151513"] because it is exposed to package 'javax.el' from resources com.ibm.websphere.javaee.el.2.2 [osgi.identity; osgi.identity="com.ibm.websphere.javaee.el.2.2"; type="osgi.bundle"; version:Version="1.0.23.201810151424"; singleton:="true"] and com.ibm.websphere.javaee.el.3.0 [osgi.identity; osgi.identity="com.ibm.websphere.javaee.el.3.0"; type="osgi.bundle"; version:Version="1.0.23.201810151424"; singleton:="true"] via two dependency chains.

Chain 1:
  com.ibm.ws.jsp.factories [osgi.identity; osgi.identity="com.ibm.ws.jsp.factories"; type="osgi.bundle"; version:Version="1.0.23.201810151513"]
    import: (&(osgi.wiring.package=javax.el)(&(version>=2.2.0)(!(version>=3.0.0))))
     |
    export: osgi.wiring.package: javax.el
  com.ibm.websphere.javaee.el.2.2 [osgi.identity; osgi.identity="com.ibm.websphere.javaee.el.2.2"; type="osgi.bundle"; version:Version="1.0.23.201810151424"; singleton:="true"]

Chain 2:
  com.ibm.ws.jsp.factories [osgi.identity; osgi.identity="com.ibm.ws.jsp.factories"; type="osgi.bundle"; version:Version="1.0.23.201810151513"]
    import: (&(osgi.wiring.package=com.ibm.ws.jsp)(&(version>=2.2.0)(!(version>=2.3.0))))
     |
    export: osgi.wiring.package=com.ibm.ws.jsp; uses:=javax.el
  com.ibm.ws.jsp [osgi.identity; osgi.identity="com.ibm.ws.jsp"; type="osgi.bundle"; version:Version="1.0.23.201810151513"]
    import: (&(osgi.wiring.package=javax.el)(version>=2.2.0))
     |
    export: osgi.wiring.package: javax.el
  com.ibm.websphere.javaee.el.3.0 [osgi.identity; osgi.identity="com.ibm.websphere.javaee.el.3.0"; type="osgi.bundle"; version:Version="1.0.23.201810151424"; singleton:="true"]

**Comment by teddyjtorres** [comment]
#build

**Comment by teddyjtorres** [comment]
The build failures were examined and are not related to the code being delivered.

**Pull request merged**